ontologies/vocabs.ttl

```diff
@@ -84,6 +84,13 @@ gist:InfrastructureType
 	rdfs:subClassOf gist:StixCategoryObject ;
 	.
 
+gist:MalwareCapability
+	a owl:Class ;
+	rdfs:subClassOf gist:StixCategoryObject ;
+	skos:definition "A Category class who's instances provide categorization by capability of malware. Describes categories of what a malware can do."^^xsd:string ;
```

Comment:
'''suggestion
skos:definition "A category indicating the nature of a malware capability i.e. what the malware can do."^^xsd:string ;
```